docs/ConfigPagesSiteBanner.md

```diff
@@ -0,0 +1,138 @@
+- Add pre-rendered config_pages site banners to the `config_page_alerts` array for template rendering
+- Support multiple paragraphs displaying as separate banners
+
+**Language Handling:**
+- Config_pages can be language-specific (`en`, `es`) or language-neutral (`und`)
+- Language-neutral config_pages appear on all language variants of the site
+- Language-specific config_pages only appear on matching language pages
+
+**Added `usa_twig_vars_theme_suggestions_paragraph_alter()` function to:**
+
+- Detect when a uswds_alert paragraph is being used in a config_pages site banner
+- Suggest the custom template for config_pages site banners
+
+#### 2. `/web/themes/custom/usagov/templates/region--header-top.html.twig`
+
+**Updated to:**
+
+- Render pre-rendered config_pages site banners (already rendered in preprocess)
+- Loop through multiple paragraphs when present
+- Remove all references to block-based site banners (completely removed)
+- No longer uses `drupal_entity()` - receives pre-rendered HTML instead
+
+#### 3. `/web/themes/custom/usagov/templates/paragraph--uswds-alert--config-pages-site-banner.html.twig`
+
+**Created custom template that:**
+
+- Inherits all styling from the standard uswds_alert paragraph template
+- Adds `data-analytics="site-banner"` attribute for analytics tracking
+- Maintains all accessibility features and USWDS styling
+- Supports all alert types (info, warning, error, success)
+- Includes proper language support for English and Spanish
+
+## How It Works
+
+### Config Pages Setup
+
+1. A config_pages entity of type 'site_banner' exists (managed via Config Pages module)
+2. This config page has a `field_uswds_paragraphs` field that can reference multiple uswds_alert paragraphs
+3. Each referenced paragraph displays as a separate banner on the site
+
+### Rendering Process
+
+1. **Region preprocessing**: `usa_twig_vars_preprocess_region()` loads the config_pages site_banner entity and extracts all referenced paragraphs
+2. **Template rendering**: `region--header-top.html.twig` loops through each config_pages alert paragraph and renders them
+3. **Template suggestion**: `usa_twig_vars_theme_suggestions_paragraph_alter()` suggests the custom template for config_pages site banners
+4. **Custom styling**: The custom template applies analytics attributes and ensures consistent styling
+
+### Benefits of This Approach
+
+1. **Centralized Management**: All site banners managed in one place via Config Pages
+2. **Multiple Banner Support**: Can display multiple alert banners simultaneously
+3. **Analytics Tracking**: Each banner includes the `data-analytics="site-banner"` attribute
+4. **Maintainability**: Uses the existing USWDS alert paragraph system
+5. **Flexibility**: Supports all USWDS alert types and configurations
+6. **Accessibility**: Maintains all accessibility features from the original implementation
+7. **Language Support**: Works with both English and Spanish content
```

Comment:
I am not seeing the language support -- I get the same banner on all pages! But we'll just need to fix that.